codegenerator/cli/npm/envio/package.json

```diff
@@ -4,55 +4,14 @@
   "private": true,
   "description": "A latency and sync speed optimized, developer friendly blockchain data indexer.",
   "bin": "./local-bin.mjs",
-  "scripts": {
-    "typecheck": "tsc --noEmit",
-    "lint": "eslint .",
-    "lint:fix": "eslint . --fix",
-    "build": "tsc",
-    "dev": "pnpm build && node lib/index.js"
-  },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/enviodev/hyperindex.git"
   },
-  "keywords": [
-    "blockchain",
-    "indexer",
-    "ethereum",
-    "fuel",
-    "data",
-    "dapp"
-  ],
```

Comment:
Any reason to remove all this metadata? I guess this was initially added for SEO?

Reply:
This is package.json for local development